### PR TITLE
Fixed replacement for windows URLs, issue #331

### PR DESCRIPTION
**Pull request by bneumann**
Tried it on my windows machine and it works like a charme now!

### CODE REVIEW

**Comment by bneumann** [comment]
I am not sure if the test has something to do with the change I made, but I will investigate

File: lib/http_server.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ var deviceFormatter = require('./api_formats/siren/device.siren');
 var rels = require('zetta-rels');
 
 var querytopic = require('./query_topic');
+console.log(url);
```
Comment (mdobson) [inline lib/http_server.js]: Be sure to remove this log statement when you get a chance :) Otherwise looking good pending that issue with a test.

Comment (bneumann) [inline lib/http_server.js]: yeah sorry for that, just saw it too while executing the tests and was wonder where the heck it came from :)

**Comment by bneumann** [comment]
There you go lad

**Pull request merged**